modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheConfigurationConsistencySelfTest.java

```diff
@@ -140,7 +138,7 @@ public void testCacheUtilsCheckAttributeMismatch() throws Exception {
 
         final GridStringLogger strLog = new GridStringLogger(false, log);
 
-        CU.checkAttributeMismatch(strLog, "cache", node.id(), "cacheMode", "Cache mode", LOCAL, PARTITIONED, false);
+        CU.checkAttributeMismatch(strLog, "cache", node.id(), "cacheMode", "Cache mode", REPLICATED, PARTITIONED, false);
```

Comment:
The `testCacheUtilsCheckAttributeMismatch` test validates only the `checkAttributeMismatch` method and nothing more. So, in this change we may set any value that differs from the `PARTITIONED` and it seems it will be correct for tests.

Anyway the test can be removed, however, I think in this PR we should keep it.